osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/infra/jms/core/OsgpCoreResponseMessageListener.java

```diff
@@ -32,7 +32,7 @@
 
     @Autowired
     @Qualifier("domainDistributionAutomationInboundOsgpCoreResponsesMessageProcessorMap")
-    private MessageProcessorMap messageProcessorMap;
+    private MessageProcessorMap messageProcessorMap; // XXX
```

Comment:
Why is // XXX added here?

Reply:
That's a booboo from me. Good catch. 

Removed.